pkg/types/options.go

```diff
@@ -97,14 +97,18 @@ type Options struct {
 	StoreResponse bool
```

Comment:
The code is generally well written and follows Go best practices. However, there are some minor improvements that could be made. Here are my recommendations:

1. **Error Handling** - The `ParseCustomHeaders` and `ParseHeadlessOptionalArguments` functions do not handle errors. If the user provides invalid input, the program may behave unexpectedly. Consider adding error returns to these functions and handle them appropriately.

go
func (options *Options) ParseCustomHeaders() (map[string]string, error) {
    ...
    if len(headerParts) < 2 {
        return nil, fmt.Errorf("invalid header format: %s", v)
    }
    ...
}

func (options *Options) ParseHeadlessOptionalArguments() (map[string]string, error) {
    ...
    if len(argParts) < 2 {
        return nil, fmt.Errorf("invalid argument format: %s", v)
    }
    ...
}


2. **Code Commenting** - Comments are used appropriately to explain the purpose of each struct field. However, the `NewProject`, `CrawlProject`, and `ListProject` fields are not commented. It would be better to maintain consistency by adding comments for these fields as well.

go
// NewProject opens visible browser window to login
NewProject string
// CrawlProject uses saved session data while crawling
CrawlProject string
// ListProject lists all project
ListProject bool


3. **Code Formatting** - The code is well-formatted overall. However, there is an empty line at the end of the code. It's a small detail, but removing it would make the code cleaner.